stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -124,11 +124,17 @@ spec:
             - name: "{{ $key }}-tcp"
               containerPort: {{ $key }}
               protocol: TCP
+              {{- if $.Values.controller.daemonset.useTcpHostPort }}
+              hostPort: {{ $key }}
```

Comment:
Like the options above, this should be configurable to a specific hostPort, not necessarily the same value as the `containerPort`. Eg you could add a new variable  .Values.controller.daemonset.hostPorts.tcp` could be a map of string -> int just like .Values.udp` mapping container port to hostPort.